src/main/java/io/frictionlessdata/tableschema/schema/TypeInferrer.java

```diff
@@ -36,18 +39,22 @@
     // The order in which the types will be attempted to be inferred.
     // Once a type is successfully inferred, we do not bother with the remaining types.
     private static final List<String[]> TYPE_INFERRAL_ORDER_LIST = new ArrayList<>(Arrays.asList(
-        new String[]{Field.FIELD_TYPE_GEOPOINT, Field.FIELD_FORMAT_DEFAULT},
-        new String[]{Field.FIELD_TYPE_GEOPOINT, Field.FIELD_FORMAT_ARRAY},
-        new String[]{Field.FIELD_TYPE_GEOPOINT, Field.FIELD_FORMAT_OBJECT},
-        new String[]{Field.FIELD_TYPE_DURATION, Field.FIELD_FORMAT_DEFAULT}, // No different formats, just use default.
+        new String[]{Field.FIELD_TYPE_BOOLEAN, Field.FIELD_FORMAT_DEFAULT}, // No different formats, just use default.
```

Comment:
Could you please explain why the change in the ordering of the inferral order list? Which cases would be parsed wrong?

Reply:
As in the other cases, there is nothing wrong with the current implementation and the change in the ordering shouldn't affect the results. 
The aim was to improve the performance of the parser handling first the cases that I found more often. Although I'm handling an amount of files less than representative, I found that the time-related fields were much more common than location-related fields, so I prioritized one group over the other.
Let's roll it back since it's a few lines of code and this PR was extensive enough and we can discuss this later on a new PR (hopefully with tests that hold my claims)